rskj-core/src/test/java/co/rsk/peg/BridgeSupportProcessFundsMigrationTest.java

```diff
@@ -89,15 +104,32 @@ void processFundsMigration_past_migration_age_after_rskip_357_activation_mainnet
         test_processFundsMigration(BridgeMainNetConstants.getInstance(), true, true, false);
     }
 
+    @Test
+    void processFundsMigration_past_migration_age_after_rskip_374_activation_mainnet() throws IOException {
+        test_processFundsMigration(BridgeMainNetConstants.getInstance(), true, true, true, false);
+    }
+
     private void test_processFundsMigration(
         BridgeConstants bridgeConstants,
         boolean isRskip146Active,
         boolean isRskip357Active,
+        boolean inMigrationAge
+    ) throws IOException {
```

Comment:
I wouldn't add this overload, just add the new parameter `isRskip374Active` to the original method. An alternative is to receive the activations object, whichever you prefer

Reply:
I prefer the activation object